server/migrations/versions/909b464cd79b_.py

```diff
@@ -0,0 +1,33 @@
+"""empty message
+
+Revision ID: 909b464cd79b
+Revises: 
+Create Date: 2020-04-19 11:51:50.754178
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '909b464cd79b'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('users',
```

Comment:
We would have an SSO service developed by our friends at Nancyclotep, so users are already stored on their end
Hence, we just need to reference the data with a user identifier

Reply:
Great I'll remove this table